libhif/hy-util.c

```diff
@@ -147,52 +150,28 @@ hy_detect_arch(char **arch)
 #undef MAX_ARCH_LENGTH
 
 int
-hy_split_nevra(const char *nevra, char **name, long int *epoch,
+hy_split_nevra(const char *nevra, char **name, int *epoch,
```

Comment:
Since we're still API unstable, making these types of changes now makes sense to me.